src/hub.js

```diff
@@ -654,7 +657,7 @@ function handleHubChannelJoined(entryManager, hubChannel, messageDispatch, data,
       scene.components["networked-scene"]
         .connect()
         .then(() => {
-          scene.emit("didConnectToNetworkedScene");
+          console.log("Networked scene connected");
```

Comment:
Pretty sure emitting `didConnectToNetworkedScene` here was correct. Was there a reason you needed to move it? In any case this log message should probably be wherever `didConnectToNetworkedScene` is emitted.

Reply:
That's right, we still need that event. I've separated this in two events, the connection to the networked scene and the connection to dialog. I believe we need both to grant access to the room.